### PR TITLE
Specify multiple gateways on PulsarCluster

### PR DESCRIPTION
**Pull request by ciiiii**
```hcl
resource "streamnative_pulsar_cluster" "test" {
  organization = "sndev"
  name = "test"
  instance_name = "instance-name"
  pool_member_name = "pm-name"
  location = "us-west1"
  config {
    websocket_enabled = true
    protocols {
      kafka = {
        enabled = true
      }
      mqtt = {
        enabled = false
      }
    }
  }
  endpoint_access {
    gateway = "public"
  }
  endpoint_access {
    gateway = "private"
  }
}
```

### CODE REVIEW

File: cloud/provider.go [hunk 1]
```diff
@@ -90,14 +90,14 @@ func init() {
 		"mqtt":                "Controls the mqtt protocol config of pulsar cluster",
 		"categories": "Controls the audit log categories config of pulsar cluster, supported categories: " +
 			"\"Management\", \"Describe\", \"Produce\", \"Consume\"",
-		"custom":                 "Controls the custom config of pulsar cluster",
-		"http_tls_service_url":   "The service url of the pulsar cluster, use it to management the pulsar cluster",
-		"pulsar_tls_service_url": "The service url of the pulsar cluster, use it to produce and consume message",
-		"kafka_service_url": "If you want to connect to the pulsar cluster using the kafka protocol," +
+		"custom":                  "Controls the custom config of pulsar cluster",
+		"http_tls_service_urls":   "The service url of the pulsar cluster, use it to management the pulsar cluster",
```
Comment (mitch-hamm) [inline cloud/provider.go:94]: Should we maybe update the description to indicate it's a list of strings?

Comment (ciiiii) [reply cloud/provider.go]: fixed

**Review by maxsxu — COMMENTED** [review]
This breaking changes will affect existing users since fields are renamed directly.

We need to take care of this change, or using progress strategy by using a new field and deprecate current field.

**Comment by ciiiii** [comment]
@maxsxu Ok, let me reserve the existing fields

**Review by maxsxu — APPROVED** [review]

**Review by mitch-hamm — APPROVED** [review]

**Pull request merged**